doc/source/getting_started.rst

```diff
@@ -21,21 +21,6 @@ You can clone the current repository of SpiceEV to your local machine using:
 
```

Comment:
I think the part "documentation" can be deleted, since the person reading is already in rtd.